policy/release/slsa_source_correlated.rego

```diff
@@ -0,0 +1,130 @@
+#
+# METADATA
+# title: SLSA - Verification model - Source
+# description: >-
+#   SLSA v1 verification model states:
+#
+#   "...artifacts are verified to ensure they meet the producer defined
+#   expectations of where the package source code was retrieved from..."
+#
+#   This package correlates the provided source code reference with the source
+#   code referenced in the attestation.
+#
+package policy.release.slsa_source_correlated
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+
+# METADATA
+# title: Expected source code reference not provided
```

Comment:
I think we want the title to be positive. Something like: "Expected source code reference".

Same for the `short_name`.

The reasoning is that these values are used whether or not the check fails. In contrast, the failure message should also be in the negative form since it only applies if there is indeed a failure/warning.